packages/@aws-cdk/aws-ecs-patterns/lib/base/scheduled-task-base.ts

```diff
@@ -38,6 +38,14 @@ export interface ScheduledTaskBaseProps {
    */
   readonly schedule: Schedule;
 
+  /**
+   * A name for the rule.
+   *
+   * @default - AWS CloudFormation generates a unique physical ID and uses that ID
+   * for the rule name. For more information, see Name Type.
```

Comment:
It might be nice to include the link to the docs: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-name.html